### PR TITLE
corectl 0.0.20

### PR DESCRIPTION
**Pull request by AntonioMeireles**
- supports VMs with more than 3GB of RAM now.

### CODE REVIEW

File: Library/Formula/corectl.rb [hunk 1]
```diff
@@ -28,6 +27,11 @@ def install
     args = []
     args << "VERSION=#{version}" if build.stable?
 
+    unless build.stable?
```
Comment (bfontaine) [inline Library/Formula/corectl.rb]: Use `if build.head?` here.

Comment (AntonioMeireles) [inline Library/Formula/corectl.rb]: fixed. thnks!

File: Library/Formula/corectl.rb [hunk 2]
```diff
@@ -28,6 +27,11 @@ def install
     args = []
     args << "VERSION=#{version}" if build.stable?
 
+    if build.head?
+      touch "unreleased"
+      system "git", "add", "unreleased"
+    end
```
Comment (mikemcquaid) [inline Library/Formula/corectl.rb]: What's this for?

Comment (AntonioMeireles) [inline Library/Formula/corectl.rb]: "cosmetics". i wanted homebrew blob built from HEAD to show (via `corectl version` exact same behavior as local git builds - explicitly telling people that they are running something that isn't a tagged version (by having `-unreleased` appended to version. In the Makefile i have `VERSION := $(shell git describe --abbrev=6 --dirty=-unreleased --always --tags)` but as that wasn't enough ence the above snippet.

Comment (mikemcquaid) [inline Library/Formula/corectl.rb]: I just don't understand why you need the `unreleased`; won't it not being a tagged version imply that anyway?

Comment (mikemcquaid) [inline Library/Formula/corectl.rb]: Also seems preferable to set `VERSION`.

Comment (AntonioMeireles) [inline Library/Formula/corectl.rb]: as i said _cosmetics_. I just wanted to achieve consistency independently the way things are built. just droped that snippet (what it was doing was to enforce the --dirty behaviour from the `git describe` that's in the Makefile).

File: Library/Formula/corectl.rb [hunk 3]
```diff
@@ -28,6 +27,15 @@ def install
...
     args << "VERSION=#{version}" if build.stable?
 
+    # system "make", "corectl", *args
+    # busts with "cannot load DWARF output from ""...
+    # similar to https://github.com/jwaldrip/homebrew-utils/issues/1
+    # workaround ...
+    ["TheNewNormal/libxhyve", "TheNewNormal/corectl/uuid2ip",
+     "yeonsh/go-ps"].each do |repo|
+      system "godep", "go", "install", "github.com/#{repo}"
+    end
```
Comment (mikemcquaid) [inline Library/Formula/corectl.rb]: This isn't necessary any more.

Comment (AntonioMeireles) [inline Library/Formula/corectl.rb]: ah, great. THANKS!!!! (fixed)

**Comment by apjanke** [comment]
Merged. Thank you for your contribution to Homebrew!

**Pull request closed without merging**